### DISCUSSION TITLE
Remove border on the table

### DISCUSSION THREAD

**Discussion by atans96** [palantir/blueprint#5747] [Q&A] [answered]
Is there a way to remove the border in the table column header? I want to override it with border:0 just like using table HTML tag normally but in blueprintjs it's using div tag instead. How can I override it?

**Comment by adidahiya** [comment, marked as answer]
Are you referring to the Table component from the `@blueprintjs/table` package?

If you want to apply custom styling, you'll need to inspect the rendered DOM, find the relevant CSS selectors for the "border" appearance, and override them with custom styles. There are no guarantees that this won't break table layout logic if you change the layout & sizing of the Table component.
